apps/web/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -420,12 +420,12 @@ export const Navigation = forwardRef(function Navigation<S extends SearchResult>
         >
           <Stream
             fallback={
-              <ul className="flex animate-pulse flex-row p-2 @4xl:gap-2 @4xl:p-5">
+              <ul className="flex animate-pulse flex-row gap-6 p-[12.5px]">
```

Comment:
Reason why I set `12.5px` is because the height of the header with links rendered is 73px minimum. With this we can match the exact height, but I'm open to other suggestions.

![Screenshot 2025-01-28 at 2 14 43 PM](https://github.com/user-attachments/assets/c83f60af-fab5-4408-b127-31acbdb252d5)

Reply:
I think generally, the approach with this is that the header should have a defined height, then the content inside is centered vertically. This way, instead of the height being driven by the contents, it's predictable. Now, IIRC we're letting the logo drive the height after a certain point. So we want to make sure that we're using a logo and a skeleton logo that's below that threshold so that the minimum height of the header is the driving value here

Reply:
Good callout. I updated to set a `min-height` to the header, and remove paddings from the skeletons. There will still be a jump when the logo renders, but that will be expected/we should let the user modify their min-height if their logo is too large.

Reply:
Is there a jump in our demo store? I feel like within a reasonable size of logo there should be no jump at all

Reply:
Not for us with these changes.